src/st-flash/flash.c

```diff
@@ -25,16 +25,25 @@ static void cleanup(int signum) {
     exit(1);
 }
 
-static void usage(void) {
-    puts("command line:   ./st-flash [--debug] [--reset] [--opt] [--serial <serial>] [--format <format>] [--flash=<fsize>] [--freq=<Hz>] {read|write} <path> [addr] [size]");
+static void usage(void)
+{
```

Comment:
Style (minor): Can we move the bracket { behind the previous argument?

Reply:
Sure, must have slipped in, fixed